cpp/src/layout/legacy/barnes_hut.cuh

```diff
@@ -93,17 +96,19 @@ void barnes_hut(raft::handle_t const& handle,
   rmm::device_uvector<int> d_childl((nnodes + 1) * 4, stream_view);
   // FA2 requires degree + 1
   rmm::device_uvector<int> d_massl(nnodes + 1, stream_view);
-  thrust::fill(handle.get_thrust_policy(), d_massl.begin(), d_massl.end(), 1);
+  rmm::device_uvector<edge_t> d_massl_et(nnodes + 1, stream_view);
+  thrust::fill(handle.get_thrust_policy(), d_massl_et.begin(), d_massl_et.end(), 1);
 
   rmm::device_uvector<float> d_maxxl(blocks * FACTOR1, stream_view);
   rmm::device_uvector<float> d_maxyl(blocks * FACTOR1, stream_view);
   rmm::device_uvector<float> d_minxl(blocks * FACTOR1, stream_view);
   rmm::device_uvector<float> d_minyl(blocks * FACTOR1, stream_view);
 
   // Actual mallocs
```

Comment:
This old comment is not correct anymore.  The allocation happens in the RAII container construction above.  I'd suggest just deleting the comment.

Reply:
Done.